tests/conftest.py

```diff
@@ -6,7 +6,7 @@
 
 
 def enumerate_unittests():
-    regexp = re.compile("XX\((.*?)\)")
+    regexp = re.compile(r"XX\((.*?)\)")
```

Comment:
I see deprecation warnings with every test I run because support for regex strings without the `r` prefix is soonish unsupported. It requires python 3.6, though (released almost six years ago). I know we focus on [compatibility](https://develop.sentry.dev/sdk/philosophy/#compatibility-is-king), but maybe it is less complicated here, since this is only used by devs? Fine to change this in another PR.